esmvaltool/references/woa2018.bibtex

```diff
@@ -0,0 +1,8 @@
+@book{woa2018,
+  title={World Ocean Atlas 2018. [temperature, salinity, oxygen, nutrients]},
```

Comment:
Is there a doi available for this reference?

Reply:
actually there are no DOI associated to the WOA reports for 2013v2 and 2018.